<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
<meta http-equiv="x-ua-compatible" content="IE=edge"/>
<title>Test results - Class scp.targets.vertx.unclustered.InitiatorExecutorTest</title>
<link href="../css/base-style.css" rel="stylesheet" type="text/css"/>
<link href="../css/style.css" rel="stylesheet" type="text/css"/>
<script src="../js/report.js" type="text/javascript"></script>
</head>
<body>
<div id="content">
<h1>Class scp.targets.vertx.unclustered.InitiatorExecutorTest</h1>
<div class="breadcrumbs">
<a href="../index.html">all</a> &gt; 
<a href="../packages/scp.targets.vertx.unclustered.html">scp.targets.vertx.unclustered</a> &gt; InitiatorExecutorTest</div>
<div id="summary">
<table>
<tr>
<td>
<div class="summaryGroup">
<table>
<tr>
<td>
<div class="infoBox" id="tests">
<div class="counter">8</div>
<p>tests</p>
</div>
</td>
<td>
<div class="infoBox" id="failures">
<div class="counter">1</div>
<p>failures</p>
</div>
</td>
<td>
<div class="infoBox" id="ignored">
<div class="counter">0</div>
<p>ignored</p>
</div>
</td>
<td>
<div class="infoBox" id="duration">
<div class="counter">0.235s</div>
<p>duration</p>
</div>
</td>
</tr>
</table>
</div>
</td>
<td>
<div class="infoBox failures" id="successRate">
<div class="percent">87%</div>
<p>successful</p>
</div>
</td>
</tr>
</table>
</div>
<div id="tabs">
<ul class="tabLinks">
<li>
<a href="#tab0">Failed tests</a>
</li>
<li>
<a href="#tab1">Tests</a>
</li>
<li>
<a href="#tab2">Standard error</a>
</li>
</ul>
<div id="tab0" class="tab">
<h2>Failed tests</h2>
<div class="test">
<a name="testLocalTimeOut"></a>
<h3 class="failures">testLocalTimeOut</h3>
<span class="code">
<pre>java.lang.AssertionError: expected:&lt;LOCAL_TIME_OUT&gt; but was:&lt;ACTION_SUCCEEDED&gt;
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at scp.api.InitiatorExecutorTest.testLocalTimeOut(InitiatorExecutorTest.java:130)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:86)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:49)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:69)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:48)
	at sun.reflect.GeneratedMethodAccessor7.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.gradle.messaging.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.messaging.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.messaging.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.messaging.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy2.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:105)
	at sun.reflect.GeneratedMethodAccessor6.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.gradle.messaging.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.messaging.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.messaging.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:355)
	at org.gradle.internal.concurrent.DefaultExecutorFactory$StoppableExecutorImpl$1.run(DefaultExecutorFactory.java:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</pre>
</span>
</div>
</div>
<div id="tab1" class="tab">
<h2>Tests</h2>
<table>
<thead>
<tr>
<th>Test</th>
<th>Duration</th>
<th>Result</th>
</tr>
</thead>
<tr>
<td class="success">testActionFailed</td>
<td>0.063s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">testActionSucceeded</td>
<td>0s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">testActionUnavailable</td>
<td>0.015s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">testExcessLoad</td>
<td>0.016s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">testFastInitiation</td>
<td>0.016s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="failures">testLocalTimeOut</td>
<td>0s</td>
<td class="failures">failed</td>
</tr>
<tr>
<td class="success">testRemoteTimeOut</td>
<td>0.125s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">testRemoteUnknownFailure</td>
<td>0s</td>
<td class="success">passed</td>
</tr>
</table>
</div>
<div id="tab2" class="tab">
<h2>Standard error</h2>
<span class="code">
<pre>[Test worker] INFO scp.targets.vertx.CommunicationManagerImpl - Done setting up vertx eventbus
[Test worker] INFO scp.targets.vertx.CommunicationManagerImpl - Done setting up vertx eventbus
[vert.x-eventloop-thread-0] WARN scp.impl.ExecutorInvoker - Dropping 2-th request
[Test worker] INFO scp.targets.vertx.CommunicationManagerImpl - Done setting up vertx eventbus
[vert.x-eventloop-thread-1] ERROR scp.impl.ExecutorInvoker - Failed to handle 1-th request
java.util.concurrent.ExecutionException: java.lang.RuntimeException: Don't Panic! This exception is good :)
	at java.util.concurrent.FutureTask.report(FutureTask.java:122)
	at java.util.concurrent.FutureTask.get(FutureTask.java:206)
	at scp.impl.ExecutorInvoker.invokeHandler(ExecutorInvoker.java:103)
	at scp.impl.ExecutorInvoker.execute(ExecutorInvoker.java:90)
	at scp.targets.vertx.CommunicationManagerImpl.lambda$registerExecutor$21(CommunicationManagerImpl.java:226)
	at io.vertx.core.eventbus.impl.EventBusImpl$HandlerRegistration.handleMessage(EventBusImpl.java:1121)
	at io.vertx.core.eventbus.impl.EventBusImpl$HandlerRegistration.handle(EventBusImpl.java:1115)
	at io.vertx.core.eventbus.impl.EventBusImpl.lambda$doReceive$179(EventBusImpl.java:755)
	at io.vertx.core.impl.ContextImpl.lambda$wrapTask$16(ContextImpl.java:335)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:358)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:357)
	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:112)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.RuntimeException: Don't Panic! This exception is good :)
	at scp.api.InitiatorExecutorTest$4.execute(InitiatorExecutorTest.java:161)
	at scp.api.InitiatorExecutorTest$4.execute(InitiatorExecutorTest.java:158)
	at scp.impl.ExecutorInvoker.lambda$invokeHandler$0(ExecutorInvoker.java:100)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	... 1 more
[Test worker] INFO scp.targets.vertx.CommunicationManagerImpl - Done setting up vertx eventbus
[Test worker] INFO scp.targets.vertx.CommunicationManagerImpl - Done setting up vertx eventbus
[Test worker] INFO scp.targets.vertx.CommunicationManagerImpl - Done setting up vertx eventbus
[Test worker] WARN scp.impl.InitiateRequester - Dropping 2-th initiation
[Test worker] INFO scp.targets.vertx.CommunicationManagerImpl - Done setting up vertx eventbus
[Test worker] INFO scp.targets.vertx.CommunicationManagerImpl - Done setting up vertx eventbus
[vert.x-eventloop-thread-0] WARN scp.impl.ExecutorInvoker - Timed out handling 1-th request
</pre>
</span>
</div>
</div>
<div id="footer">
<p>Generated by 
<a href="http://www.gradle.org">Gradle 2.1</a> at Dec 4, 2015 7:30:11 PM</p>
</div>
</div>
</body>